### PR TITLE
Improve the readability of the stream initialization logic

### PR DESCRIPTION
**Pull request by wplong11**
This PR worked for https://github.com/ReactorKit/ReactorKit/pull/224

### CODE REVIEW

**Review by tokijh — APPROVED** [review]
I'm concerned that the behavior changes depending on the Xcode version, but LGTM

**Comment by wplong11** [comment]
The problem you are concerned about in PR #30 is not reproduced now! Still, the test code added in the PR was not deleted. The result of this PR is the same structure as in PR #30. Public properties action and state are implemented by accessing private properties. Although I haven't been able to test it on an older version of Xcode, I think there will be no problem.
https://github.com/ReactorKit/ReactorKit/pull/30

File: Sources/ReactorKit/Reactor.swift [hunk 1]
```diff
@@ -71,60 +71,52 @@ public protocol Reactor: AnyObject {
...
+}
+
 // MARK: - Default Implementations
 
 extension Reactor {
-  private var _action: ActionSubject<Action> {
-    if isStubEnabled {
-      return stub.action
+  private var streams: ReactorStreams<Action, State> {
+    if (isStubEnabled) {
```
Comment (OhKanghoon) [inline Sources/ReactorKit/Reactor.swift:93]:
```suggestion
    if isStubEnabled {
```
swift prefer convention that does not use brackets in the conditional state.

Comment (wplong11) [reply Sources/ReactorKit/Reactor.swift]: Modified! 👍

**Comment by wplong11** [comment]
@tokijh @OhKanghoon 
https://github.com/ReactorKit/ReactorKit/pull/223#issuecomment-1940071358

I have excluded the changes related to #30 from this PR and will create a separate PR to follow. I don't think there is a problem, but it hasn't been verified very strictly.

**Comment by OhKanghoon** [comment]
> @tokijh @OhKanghoon [#223 (comment)](https://github.com/ReactorKit/ReactorKit/pull/223#issuecomment-1940071358)
> 
> I have excluded the changes related to #30 from this PR and will create a separate PR to follow. I don't think there is a problem, but it hasn't been verified very strictly.

It is predicted that the problem will not occur because the unit test related to #30 remains.

**Review by OhKanghoon — APPROVED** [review]
Thanks 👍

**Pull request merged**